spec.html

```diff
@@ -23766,15 +23743,14 @@ <h1>GetExportedNames ( [ _exportStarSet_ ] ) Concrete Method</h1>
 
         <emu-clause id="sec-resolveexport">
           <h1>ResolveExport ( _exportName_ [ , _resolveSet_ ] ) Concrete Method</h1>
-          <p>The ResolveExport concrete method of a Source Text Module Record implements the corresponding Module Record abstract method.</p>
+          <p>The ResolveExport concrete method of a Source Text Module Record _module_ takes argument _exportName_ (a String) and optional argument _resolveSet_.</p>
           <p>ResolveExport attempts to resolve an imported binding to the actual defining module and local binding name. The defining module may be the module represented by the Module Record this method was invoked on or some other module that is imported by that module. The parameter _resolveSet_ is used to detect unresolved circular import/export paths. If a pair consisting of specific Module Record and _exportName_ is reached that is already in _resolveSet_, an import circularity has been encountered. Before recursively calling ResolveExport, a pair consisting of _module_ and _exportName_ is added to _resolveSet_.</p>
           <p>If a defining module is found, a ResolvedBinding Record { [[Module]], [[BindingName]] } is returned. This record identifies the resolved binding of the originally requested export, unless this is the export of a namespace with no local binding. In this case, [[BindingName]] will be set to *"\*namespace\*"*. If no definition was found or the request is found to be circular, *null* is returned. If the request is found to be ambiguous, the string *"ambiguous"* is returned.</p>
-          <p>This abstract method performs the following steps:</p>
+          <p>This concrete method performs the following steps when called:</p>
```

Comment:
```suggestion
          <p>The ResolveExport concrete method performs the following steps when called:</p>
```

Reply:
It's a bit misleading to say "the ResolveExport concrete method", because it suggests there's only one, which won't be true once there's another subtype of Module Record (e.g., if 'JSON modules' is merged). We could say `The ResolveExport concrete method of a Source Text Module Record performs ...`, but I'm wondering why either is preferable to `This concrete method performs...`: it's not like there's any *other* method mentioned in the preceding paragraphs.